app/controllers/benefits/assets/AssetsOrAssetTransfersBenefitsController.scala

```diff
@@ -84,8 +85,7 @@ class AssetsOrAssetTransfersBenefitsController @Inject()(implicit val cc: Messag
                 data.hasPriorBenefits
               )(errorHandler.internalServerError()) {
                 val nextPage = if (yesNo) {
-                  //TODO got to Assets question
-                  CheckYourBenefitsController.show(taxYear, employmentId)
+                  AssetsBenefitsController.show(taxYear, employmentId)
                 } else {
                   //TODO next section
```

Comment:
I believe this comment can be removed now as it needs to go to check your benefits controller.